run.sh

```diff
@@ -61,19 +68,29 @@ execute_model() {
   # Go to repo root (where run.sh lives).
   cd "$(dirname "$0")"
 
-  echo ">>> Generating state and county models to ${API_OUTPUT_DIR}"
-  # TODO(#148): We need to clean up the output of these scripts!
-  python pyseir/cli.py build-all --output-dir="${API_OUTPUT_DIR}" | tee "${API_OUTPUT_DIR}/stdout.log"
-
-  # Move state output to the expected location.
-  mkdir -p ${API_OUTPUT_DIR}/
-
-  # Capture all the PDFs pyseir creates in output/pyseir since they are
-  # extremely helpful for debugging / QA'ing the model results.
-  echo ">>> Generating pyseir.zip from PDFs in output/pyseir."
-  pushd output
-  zip -r "${API_OUTPUT_DIR}/pyseir.zip" pyseir/* -i '*.pdf'
-  popd
+  if [ ! -z "$PYSEIR_ARTIFACT_SNAPSHOT" ]; then
+    echo ">>> Downloading state and county models from existing snapshot ${PYSEIR_ARTIFACT_SNAPSHOT}."
+    ./run.py utils download-model-artifact "${PYSEIR_ARTIFACT_SNAPSHOT}" --output-dir=${API_OUTPUT_DIR%/*}
```

Comment:
I guess `${API_OUTPUT_DIR%/*}` is the parent directory of API_OUTPUT_DIR but that wasn't obvious to me, so I'd probably create a variable for that:

```
API_OUTPUT_PARENT="${API_OUTPUT_DIR%/*}"
```

or similar.  That said, I wonder why we are putting it there...  Would it make sense to just use `/tmp/` or something since we're going to move the files afterwards anyway?

Reply:
That said, if you don't feel like changing it and needing to retest it, feel free to just keep this all as-is.  Maybe add a `TODO(sean): Might be simpler to just download the model results to /tmp/` comment to revisit next time we're messing with this code.